src/locales/en.json

```diff
@@ -40,8 +40,8 @@
 
   "InstallationPage": {
     "title": "Install Cozy extension",
-    "descriptionStart": "Add",
-    "descriptionEnd": "to your browser to automatically fill and save your passwords while browsing",
+    "descriptionStart": "Add the",
+    "descriptionEnd": "to your browser. This installation is necessary to allow you to automatically fill in and save your passwords while browsing.",
```

Comment:
Nit: AMHA, when we need to fill some part of a sentence, we should not have several parts in the translation files, but a single one, here "description" with a `%{variable}` in it.

Reply:
The problem here is that we want to insert a react component in the middle of the sentence.